docs/integrations/wallets.md

```diff
@@ -0,0 +1,108 @@
+</a>
+
+For more examples, click [here](https://github.com/lidofinance/referral-program-integration-examples/tree/main/examples/banners).
+
+### Embedding Staking Widget
+
+If you don't want users to leave your app to stake, you can embed Staking Widget via the `iframe` HTML tag and use your referral link as the source.
+
+Example code,
+```html
+<iframe
+    src="https://stake.lido.fi/?ref=0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B"
+    title="Lido Staking App"
+    height="500"
+    width="100%"
+/>
+```
+
+Preview,
+<iframe
```

Comment:
in my preview of the file - the iframe is displayed in full working order.
It seems to me that this will be a problem because everyone who opens the guide will load the widget in the iframe

<img width="1139" alt="image" src="https://user-images.githubusercontent.com/20888859/200775711-958e8863-66b1-4387-8fc1-b84799d5abba.png">